README.md

```diff
@@ -1 +1,31 @@
-# tapyrusjs-wallet
+# Tapyrus Wallet for Javascript
+
+## Install
+
+```
+npm install chaintope/tapyrusjs-wallet
+```
+
+## Usage
+
+### Cordova Plugin
+
+```
+const tapyrus = require("tapyrusjs-lib");
+const wallet = require("tapyrusjs-wallet");
+
+const alice = new wallet.Wallet.BaseWallet();
+
+alice.keyStore = new wallet.KeyStore.CordovaKeyStore();
+alice.import("tprv8ZgxMBicQKsPd7Uf69XL1XwhmjHopUGep8GuEiJDZmbQz6o58LninorQAfcKZWARbtRtfnLcJ5MQ2AtHcQJCCRUcMRvmDUjyEmNUWwx8UbK");
```

Comment:
This key looks `dev` key value of extended private key. How to switch mode (`prod` or `dev`)?

Reply:
Add class to configure network mode.
(920f2adcb0464559c162eefa179b364280a5b8ed)